src/pages/Blog/BlogItem/loader.ts

```diff
@@ -0,0 +1,34 @@
+import { json, LoaderFunction } from "react-router-dom";
+import { isPost } from "./BlogItem";
+export const loader: LoaderFunction = async ({ params }) => {
```

Comment:
we will need some kind of better prettier config, because i really need one line beneath the imports

Reply:
I agree